Dockerfile

```diff
@@ -1,11 +1,12 @@
 ARG VARIANT=2.6
+
 FROM mcr.microsoft.com/vscode/devcontainers/ruby:${VARIANT}
 
 # Install node
-COPY .nvmrc /tmp/.nvmrc
+ARG NODE_VERSION="12.1"
```

Comment:
Why is this a build arg?

Reply:
because a constructor inline was not working, but I will try another path and revert this.